docs/Upgrade.md

```diff
@@ -0,0 +1,45 @@
+# Upgrade
+
+## 1. Overview
+
+This document contains pointer to instructions on how to upgrade the
+Zonemaster::Backend component. An upgrade usually consist of an upgrade script
+to upgrade the database, and instructions to install new dependencies.
+
+
+## 2. Upgrading Zonemaster::Backend
+
+To upgrade Zonemaster::Backend perform the following tasks:
+
+  1. stop the `zm-rpcapi` and `zm-testagent` daemons
+  2. install the latest version from `cpanm` with `cpanm Zonemaster::Backend`
```

Comment:
This will, I think, automatically upgrade Zonemaster::LDNS and Zonemaster::Engine too, which is good in one way, but if some installation parameter is used for that OS, then it is maybe less good.

Reply:
I understand the use case you are describing, however I don't see how this use use case was handled before.

Are you suggesting that we should consider adding a note for this specific use case? Or can we consider that if someone is in this use case that means that they have already moved apart from the installation instructions, so we can infer that they can continue to tweak the instructions?

Reply:
Maybe a reference that Engine should be updated first, with a link to the Engine installation document (in the future maybe an Engine upgrade instruction).

Besides the possible parameters for Zonemaster::LDNS there is also the possibility that there are new dependencies for Zonemaster::LDNS or Zonemaster::Engine that should be installed first. (Zonemaster::LDNS is handled together with Zonemaster::Engine.)